test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -146,6 +147,37 @@ call_log_manager::process_call_log_entry(json call_log_entry)
             }
             break;
         }
+        case api_method::query_timestamp: {
+            /*
+             * Not having a valid connection is a fatal error since no other operations can happen
+             * without a connection.
+             */
+            if (_conn == nullptr)
+                throw std::runtime_error(
+                  "Could not query the timestamp as connection does not exist");
+
+            /* Convert the config char * to a string object. */
+            std::string config = call_log_entry["input"]["config"].get<std::string>();
+            std::string hex_ts, hex_ts_expected;
```

Comment:
Declare those where you use them

Reply:
Updated in [4a62e95](https://github.com/wiredtiger/wiredtiger/pull/8145/commits/4a62e95768f31c15421f190f70c93abf28157f99)